Documentation/project-docs/uwp-compat.md

```diff
@@ -0,0 +1,489 @@
+# Behavior change between .NET Core and UWP
+
+This document contains list of not supported APIs on UWP but supported on .NET Core
+
+## Notes
+- This document is for preview only and was semi-generated, information may be incomplete or may contain errors
```

Comment:
Would it be better for the doc to live on the wiki, next to https://github.com/dotnet/corefx/wiki/ApiCompat ?